build/istanbul-config.js

```diff
@@ -1,24 +1,20 @@
 'use strict';
 
-var path = require('path');
+var unitTestCoverageDirectory = './coverage/unit';
+var unitTestCoverageReportHtmlFile = './coverage/unit/index.html';
 
```

Comment:
Lol! I do that so much.. As I imagine will be the case with other things pulled from the generator, it will require a fix in two places (the updated source for the generator code itself, and the template content it came from)